src/sentry/apidocs/api_pagination_allowlist_dont_modify.py

```diff
@@ -0,0 +1,140 @@
+    "/api/0/organizations/{organization_slug}/api-keys/",
+    "/api/0/organizations/{organization_slug}/auth-providers/",
+    "/api/0/organizations/{organization_slug}/config/integrations/",
+    "/api/0/organizations/{organization_slug}/config/repos/",
+    "/api/0/organizations/{organization_slug}/sdk-updates/",
+    "/api/0/organizations/{organization_slug}/sdks/",
+    "/api/0/organizations/{organization_slug}/events/",
+    "/api/0/organizations/{organization_slug}/events-stats/",
+    "/api/0/organizations/{organization_slug}/metrics-estimation-stats/",
+    "/api/0/organizations/{organization_slug}/events-facets/",
+    "/api/0/organizations/{organization_slug}/events-span-ops/",
+    "/api/0/organizations/{organization_slug}/events-spans-stats/",
+    "/api/0/organizations/{organization_slug}/events-root-cause-analysis/",
+    "/api/0/organizations/{organization_slug}/spans-samples/",
+    "/api/0/organizations/{organization_slug}/metrics-compatibility-sums/",
+    "/api/0/organizations/{organization_slug}/missing-members/",
+    "/api/0/organizations/{organization_slug}/events-vitals/",
+    "/api/0/organizations/{organization_slug}/events-has-measurements/",
+    "/api/0/organizations/{organization_slug}/events-trends-stats/",
+    "/api/0/organizations/{organization_slug}/issues/",
+    "/api/0/organizations/{organization_slug}/issues-stats/",
+    "/api/0/organizations/{organization_slug}/integrations/",
+    "/api/0/organizations/{organization_slug}/integrations/{integration_id}/repos/",
+    "/api/0/organizations/{organization_slug}/integrations/{integration_id}/serverless-functions/",
+    "/api/0/organizations/{organization_slug}/notifications/available-actions/",
+    "/api/0/organizations/{organization_slug}/monitors-stats/",
+    "/api/0/organizations/{organization_slug}/monitors/{monitor_slug}/stats/",
+    "/api/0/organizations/{organization_slug}/recent-searches/",
+    "/api/0/organizations/{organization_slug}/searches/",
+    "/api/0/organizations/{organization_slug}/members/{member_id}/unreleased-commits/",
+    "/api/0/organizations/{organization_slug}/processingissues/",
+    "/api/0/organizations/{organization_slug}/projects/",
+    "/api/0/organizations/{organization_slug}/plugins/",
+    "/api/0/organizations/{organization_slug}/plugins/configs/",
+    "/api/0/organizations/{organization_slug}/release-threshold-statuses/",
+    "/api/0/organizations/{organization_slug}/releases/{version}/files/",
+    "/api/0/organizations/{organization_slug}/releases/{version}/previous-with-commits/",
+    "/api/0/organizations/{organization_slug}/user-teams/",
+    "/api/0/organizations/{organization_slug}/users/",
+    "/api/0/organizations/{organization_slug}/org-auth-tokens/",
+    "/api/0/organizations/{organization_slug}/stats/",
+    "/api/0/organizations/{organization_slug}/tags/",
+    "/api/0/organizations/{organization_slug}/environments/",
+    "/api/0/organizations/{organization_slug}/replays/",
```

Comment:
it sems like the replays endpoint does use self.paginate? what's going on with that?
https://github.com/getsentry/sentry/blob/3ec7178697b63cddea6ebc8cb10968df127e4d00/src/sentry/api/urls.py#L1891
https://github.com/getsentry/sentry/blob/3ec7178697b63cddea6ebc8cb10968df127e4d00/src/sentry/replays/endpoints/organization_replay_index.py#L101

Reply:
Good catch! The regex expression currently does not handle nested function definitions, which is why it didn't find the self.paginate.